modules/storage/local.go

```diff
@@ -36,7 +36,7 @@ func NewLocalStorage(ctx context.Context, config *setting.Storage) (ObjectStorag
 	}
 
 	if config.TemporaryPath == "" {
-		config.TemporaryPath = filepath.Join(config.Path, "tmp")
+		config.TemporaryPath = filepath.Join(setting.TempPath, filepath.Base(config.Path))
```

Comment:
Why `filepath.Base`?

Reply:
It's a bug. We need an unique path under the tmppath.

b88c3a6af3